pkg/kubelet/types/constants.go

```diff
@@ -16,8 +16,22 @@ limitations under the License.
 
 package types
 
+import "runtime"
+
 const (
-	PodInfraContainerImage = "gcr.io/google_containers/pause:2.0"
+
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
+
+	DefaultPodInfraContainerImageName = "gcr.io/google_containers/pause"
+	DefaultPodInfraContainerImageVersion = "2.0"
 )
+
+// Returns the arch-specific pause image that kubelet use as the default
+func GetDefaultPodInfraContainerImage() string {
+	if runtime.GOARCH == "amd64" {
+		return DefaultPodInfraContainerImageName + ":" + DefaultPodInfraContainerImageVersion
+	} else {
+		return DefaultPodInfraContainerImageName +  "-" + runtime.GOARCH + ":" + DefaultPodInfraContainerImageVersion
```

Comment:
@mikedanese What do you have to convince me of?   I wasn't involved in any of this naming.  My general guidance here is to follow a (sane) precedent and to be consistent with other projects.